newIDE/app/src/GameDashboard/PublicGamePropertiesDialog.js

```diff
@@ -11,12 +11,15 @@ import {
 import FlatButton from '../UI/FlatButton';
 import Dialog from '../UI/Dialog';
 import { type PublicGame } from '../Utils/GDevelopServices/Game';
+import AuthenticatedUserContext from '../Profile/AuthenticatedUserContext';
 
 /**
  * Changes that are not stored in the Project.
  */
 export type PartialGameChange = {|
-  ownerIds?: Array<string>,
+  ownerIds: Array<string> | null,
+  userSlug: string | null,
+  gameSlug: string | null,
```

Comment:
Any reason not to do that: 

```suggestion
  ownerIds?: Array<string>,
  userSlug?: string,
  gameSlug?: string,
```

A property with a `?` after its name indicated that you're not forced to specify it. This means it will be `undefined`, and the semantic is "I've not specified it so don't even attempt to change/modify it".
While a `null` has a slighlty different meaning. This can be understood as "I specified this property, so I probably want to set it to null, i.e: remove it."

Reply:
As it's used at 2 places, I was worried that optional attributes could lead to forbid to update the UI for the browser version.
But, you're right, null has a meaning, I will use optional attributes.

Reply:
Totally, you're right that the big default of allowing to not specify an attribute is that.. well you can forget to specify it :) 
What I usually do is that I set the property as mandatory (I remove the `?`), I check everywhere in the code where Flow is complaining, and if it's ok I put back the `?`. Not ideal but better than nothing!